<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTextEdit" name="textArea">
      <property name="cursorWidth">
       <number>1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_new_file"/>
    <addaction name="action_open"/>
    <addaction name="action_save"/>
    <addaction name="action_close"/>
    <addaction name="action_quit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="action_copy"/>
    <addaction name="action_cut"/>
    <addaction name="action_paste"/>
    <addaction name="action_select_all"/>
    <addaction name="action_clear_Selection"/>
    <addaction name="action_undo"/>
    <addaction name="action_redo"/>
   </widget>
   <widget class="QMenu" name="menuSearch">
    <property name="title">
     <string>Search</string>
    </property>
    <addaction name="action_find"/>
    <addaction name="action_find_and_replace"/>
    <addaction name="action_next_search_result"/>
    <addaction name="action_previous_search_result"/>
   </widget>
   <widget class="QMenu" name="menuFont">
    <property name="title">
     <string>Text</string>
    </property>
    <addaction name="action_choose_font"/>
    <addaction name="action_choose_font_color"/>
    <addaction name="action_choose_background_color"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_about_program"/>
    <addaction name="action_about_Qt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuSearch"/>
   <addaction name="menuFont"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_open"/>
   <addaction name="action_new_file"/>
   <addaction name="action_save"/>
   <addaction name="separator"/>
   <addaction name="action_copy"/>
   <addaction name="action_cut"/>
   <addaction name="action_paste"/>
   <addaction name="action_select_all"/>
   <addaction name="action_clear_Selection"/>
   <addaction name="action_undo"/>
   <addaction name="action_redo"/>
   <addaction name="separator"/>
   <addaction name="action_find"/>
   <addaction name="action_find_and_replace"/>
   <addaction name="action_previous_search_result"/>
   <addaction name="action_next_search_result"/>
   <addaction name="separator"/>
   <addaction name="action_choose_font"/>
   <addaction name="action_choose_font_color"/>
   <addaction name="action_choose_background_color"/>
   <addaction name="separator"/>
   <addaction name="action_about_program"/>
   <addaction name="action_about_Qt"/>
   <addaction name="separator"/>
   <addaction name="action_quit"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_about_program">
   <property name="icon">
    <iconset>
     <normaloff>:/img/icons2/about.png</normaloff>:/img/icons2/about.png</iconset>
   </property>
   <property name="text">
    <string>About program</string>
   </property>
  </action>
  <action name="action_about_Qt">
   <property name="icon">
    <iconset>
     <normaloff>:/img/icons2/qt.png</normaloff>:/img/icons2/qt.png</iconset>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="action_open">
   <property name="icon">
    <iconset>
     <normaloff>:/img/icons2/open3.png</normaloff>:/img/icons2/open3.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_save">
   <property name="icon">
    <iconset>
     <normaloff>:/img/icons2/Save.png</normaloff>:/img/icons2/Save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_close">
   <property name="icon">
    <iconset>
     <normaloff>:/img/icons2/close_file.svg</normaloff>:/img/icons2/close_file.svg</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="icon">
    <iconset>
     <normaloff>:/img/icons2/Quit.png</normaloff>:/img/icons2/Quit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_copy">
   <property name="icon">
    <iconset>
     <normaloff>:/img/icons2/copy.png</normaloff>:/img/icons2/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_cut">
   <property name="icon">
    <iconset>
     <normaloff>:/img/icons2/cut.png</normaloff>:/img/icons2/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="action_paste">
   <property name="icon">
    <iconset>
     <normaloff>:/img/icons2/paste.png</normaloff>:/img/icons2/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="action_select_all">
   <property name="icon">
    <iconset>
     <normaloff>:/img/icons2/selectAll.png</normaloff>:/img/icons2/selectAll.png</iconset>
   </property>
   <property name="text">
    <string>Select all</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="action_clear_Selection">
   <property name="icon">
    <iconset>
     <normaloff>:/img/icons2/clear_selection.png</normaloff>:/img/icons2/clear_selection.png</iconset>
   </property>
   <property name="text">
    <string>Clear Selection</string>
   </property>
  </action>
  <action name="action_new_file">
   <property name="icon">
    <iconset>
     <normaloff>:/img/icons2/new_file.png</normaloff>:/img/icons2/new_file.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_choose_font">
   <property name="icon">
    <iconset>
     <normaloff>:/img/icons2/choose_font.png</normaloff>:/img/icons2/choose_font.png</iconset>
   </property>
   <property name="text">
    <string>Choose font</string>
   </property>
  </action>
  <action name="action_choose_font_color">
   <property name="icon">
    <iconset>
     <normaloff>:/img/icons2/font_color_2.png</normaloff>:/img/icons2/font_color_2.png</iconset>
   </property>
   <property name="text">
    <string>Choose font color</string>
   </property>
  </action>
  <action name="action_choose_background_color">
   <property name="icon">
    <iconset>
     <normaloff>:/img/icons2/color_options.png</normaloff>:/img/icons2/color_options.png</iconset>
   </property>
   <property name="text">
    <string>Choose background color</string>
   </property>
  </action>
  <action name="action_find">
   <property name="icon">
    <iconset>
     <normaloff>:/img/icons2/search.png</normaloff>:/img/icons2/search.png</iconset>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
  </action>
  <action name="action_find_and_replace">
   <property name="icon">
    <iconset>
     <normaloff>:/img/icons2/replace.png</normaloff>:/img/icons2/replace.png</iconset>
   </property>
   <property name="text">
    <string>Find and replace</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="action_next_search_result">
   <property name="icon">
    <iconset>
     <normaloff>:/img/icons2/find_next.png</normaloff>:/img/icons2/find_next.png</iconset>
   </property>
   <property name="text">
    <string>Next search result</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Right</string>
   </property>
  </action>
  <action name="action_previous_search_result">
   <property name="icon">
    <iconset>
     <normaloff>:/img/icons2/find_previous.png</normaloff>:/img/icons2/find_previous.png</iconset>
   </property>
   <property name="text">
    <string>Previous search result</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Left</string>
   </property>
  </action>
  <action name="action_undo">
   <property name="icon">
    <iconset>
     <normaloff>:/img/icons2/undo.png</normaloff>:/img/icons2/undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="action_redo">
   <property name="icon">
    <iconset>
     <normaloff>:/img/icons2/redo.png</normaloff>:/img/icons2/redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
